idp-service/README.md

```diff
@@ -1,8 +1,63 @@
 ## Identity Provider Service
 
+IdP OIDC service, provides endpoints to 
+1. Generate auth-code 
+2. Fetch id-token and access-token
+3. Fetch userinfo (KYC)
+
 ![](/docs/IdP-service-basic-flow.png)
 
 
+## Caching details
+
+IdP-UI transaction
+
+| Endpoint    | Cache                              | Evict                              |
+|-------------|------------------------------------|------------------------------------|
+| oauthDetails | preauthsessions (k: transactionId) |                                    |
+| authenticate | authenticated (k: transactionId)   | preauthsessions (k: transactionId) |
+| authCode    | consented (k: code)                | authenticated (k: transactionId)   |
+| token       | kyc   (k: accessTokenHash)         | consented  (k: code)           |
+
+
+Linked transactions
+
+| Endpoint           | Cache                                                     | Evict                              |
+|-----------------|-----------------------------------------------------------|------------------------------------|
+| oauthDetails    | preauthsessions (k: transactionId)                        |                                    |
+| linkTransaction | linkedsessions (k: linkTransactionId)                     | preauthsessions (k: transactionId) |
```

Comment:
we should have the link code also rt ?

Reply:
If we are using linkCodeHash, then the variable and cache names also can we call it as respective hash ?

Reply:
renamed the cache to linkCodeHash.